package.json

```diff
@@ -3,13 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.0",
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "@types/react-redux": "^7.1.23",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.2",
     "react-scripts": "5.0.0",
+    "uuid": "^8.3.2",
```

Comment:
- [ ] Your project is compiling with errors, kindly fix them:
![Screenshot from 2022-03-29 21-37-02](https://user-images.githubusercontent.com/62705947/160694511-8464bdbc-9643-425b-8b8f-6ca3a9837916.png)